src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -73,4 +75,13 @@ public static MedicalHistory getSampleMedicalHistory() {
         MedicalHistory test = new MedicalHistory(mh);
         return test;
     }
+    //@@ GAO JIAXIN
+    public static VisitorList getSampleVisitorList() {
+        String[] visitors = {"jack", "daniel"};
+        ArrayList<Visitor> vl = Arrays.stream(visitors)
```

Comment:
Violation of Interface Segregation Principle.
Consider using List<> instead of ArrayList<>.